### PR TITLE
make ocis-hello compile and run with ocis 3.0.0

### PR DESCRIPTION
**Pull request by individual-it**
This PR makes the ocis-hello extension work with the current version of ocis
- update dependencies
- update ocis as dependency (specific branch+commit because the master branch contains files with invalid [for go] filenames and the tag does not work ether)
- update documentation
- change UI tests to work with playwright
- fix CI

### CODE REVIEW

**Comment by mmattel** [comment]
Pls revert/remove the text introduced in #138 
See comment `The ocis-hello example is outdated and does not work - this is now noted.`

**Comment by individual-it** [comment]
@mmattel I've reverted the changes and fixed the link in [1fdea13](https://github.com/owncloud/ocis-hello/pull/140/commits/1fdea13ad52715bab6ab956edbda2d4197cfc3b8)
But I think that warning message never made it into the public docs: https://owncloud.dev/extensions/ocis_hello/
maybe because CI was brocken

**Review by mmattel — APPROVED** [review]
LGTM from a docs pov

**Review by phil-davis — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -1,32 +1,191 @@
...
-	github.com/golang/protobuf v1.5.2
+	github.com/go-chi/chi/v5 v5.0.8
+	github.com/go-chi/render v1.0.2
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.2
 	github.com/micro/cli/v2 v2.1.2
 	github.com/oklog/run v1.1.0
 	github.com/openzipkin/zipkin-go v0.2.5
-	github.com/owncloud/ocis v1.11.0
-	github.com/prometheus/client_golang v1.11.0
+	github.com/owncloud/ocis/v2 v2.0.1-0.20230529065344-fd4301b0d71e
```
Comment (individual-it) [inline go.mod:18]: This is a specific commit on an own branch inside ocis that does not have the `tests/acceptance/filesForUpload/'single'quotes.txt` file

The issue is that when I add ocis as a dependency 
`go get  github.com/owncloud/ocis/v2@v3.0.0-rc.4` gives me
`go: github.com/owncloud/ocis/v2@v3.0.0-rc.4: invalid version: module path includes a major version suffix, so major version must match`

And
`go get github.com/owncloud/ocis/v2@7094891f4de381102b05c6503751dc85d82c0782 #latest commit` gives me
```
go: downloading github.com/owncloud/ocis/v2 v2.0.1-0.20230530000306-7094891f4de3
go: github.com/owncloud/ocis/v2@7094891f4de381102b05c6503751dc85d82c0782: create zip: tests/acceptance/filesForUpload/'single'quotes.txt: malformed file path "tests/acceptance/filesForUpload/'single'quotes.txt": invalid char '\''
tests/acceptance/filesForUpload/new-'single'quotes.txt: malformed file path "tests/acceptance/filesForUpload/new-'single'quotes.txt": invalid char '\''
```

I found somewhere a bug-report for that, go can only work with certain file-names

The only way to do it I found is to make a commit on a new branch in ocis that does not have the problematic file. But what is the "right" thing to do?

**Comment by kulmann** [comment]
Out of curiosity, where did the idea come from to make `ocis-hello` great/work again? I don't see a ticket for it and regarding our ocis GA our product management accepted the fact that it was broken. It doesn't have much value in terms of showcasing our extension systems in ocis and web (it's a huge bunch of boilerplate code). I appreciate your effort but I wouldn't have recommended to invest the time.

**Comment by individual-it** [comment]
@kulmann We from JankariTech want to learn how ocis-extensions can be done and what the possibilities are. For that it looked like a good idea to first make ocis-hello work again. This is an unpaid open-source contribution.
The background behind all of that is, that we are hoping to integrate ocis with other products.

**Review by phil-davis — APPROVED** [review]

**Review by micbar — APPROVED** [review]

**Comment by kulmann** [comment]
> @kulmann We from JankariTech want to learn how ocis-extensions can be done and what the possibilities are. For that it looked like a good idea to first make ocis-hello work again. This is an unpaid open-source contribution.
> The background behind all of that is, that we are hoping to integrate ocis with other products.

wow, didn't expect that... just wow! thank you!

**Pull request merged**